ballsdex/packages/trade/cog.py

```diff
@@ -419,6 +429,62 @@ async def history(
         pages = Pages(source=source, interaction=interaction)
         await pages.start()
 
+    @coins.command(name="add")
+    async def coins_add(self, interaction: discord.Interaction, amount: int):
+        """
+        Add coins to your trade proposal
+        """
+        trade, trader = await self.check_trade_errors(interaction)
+        if not trade or not trader:
+            return
+
+        if amount < 1:
+            await interaction.response.send_message(
+                "`amount` must be superior or equal to 1.", ephemeral=True
+            )
+            return
+        if amount > await trader.fetch_player_coins():
```

Comment:
```suggestion
        if amount > await trader.fetch_player_coins() - trader.coins:
```
otherwise you can exploit the commnand